### ISSUE TITLE
Should I update all pre-render HTML to Sever?

### ISSUE THREAD

**Issue by Tays-s** [chrisvfritz/prerender-spa-plugin#299]
Q1: 
After generated local pre-render HTML files, should I update these pre-render HTML files to server?

Q2: 
if index has parameter,  the parameter can be changed dynamically?

**Comment by Tribex** [comment]
Yes, you should upload them to your server. Parametric routes cannot be prerendered dynamically, unfortunately.

**Comment by Tays-s** [comment]
> Yes, you should upload them to your server. Parametric routes cannot be prerendered dynamically, unfortunately.
Okey, thank you for your comment
